lib/Net/Kubernetes.pm

```diff
@@ -172,6 +177,57 @@ sub get_namespace {
     }
 }
 
+sub create_namespace {
+    my ($self, $namespace) = @_;
+
+    if (!defined $namespace || !length $namespace) {
+        Throwable::Error->throw(message => '$namespace cannot be null');
+    }
+
+    my $namespace_path = $self->get_resource_path('namespace');
+    my $res = $self->ua->request($self->create_request(
+        POST => $namespace_path, 
+        undef, $self->json->encode({ 
```

Comment:
Just a style nit, but maybe a newline between `undef` and the json body would make this not read quite so awkwardly.